templates/index.html

```diff
@@ -211,15 +210,15 @@ <h3>Discord</h3>
         </p>
 
         <p>
-          <a class="button-small" href="http://kakaporecovery.org.nz/sirocco/">Donate To Keep Them Partying</a>
+          <a class="button-small" href="https://www.doc.govt.nz/kakapo-recovery">Donate To Keep Them Partying</a>
```

Comment:
Domain `kakaporecovery.org.nz` doesn't seem to have an https certificate, but the link redirects to `www.doc.govt.nz` anyway, which does.